text/0000-ember-new-lang.md

```diff
@@ -1,27 +1,69 @@
 >
 > **Source: [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)**
 
-When the language of the page cannot be identified, the integrity of the above information cannot be guaranteed. In this context, it is the users of screen readers, braille translation software, and similar assitive technologies for whom valid page language specifications provide the greatest improvements to user experience and technical operation. It is, however, extremely important to note that that although digital accessibility concerns are the primary motivators for developing formalized page language specifications, achieving WCAG SC-3.1.1 should certainly be considered a global application improvement. The following list contains application use cases that all benefit from having a valid page language specified, but are not strictly tied to digital accessibility requirements or assistive technology:
+When the language of the page cannot be identified, the integrity of the above information cannot be guaranteed. 
+Consider the following use case:
+
+the application developer is unaware that Ember now includes the lang attribute
+the application does not require internationalization
+the application's content is in a language that is not English
+an end-user with a screen reader turned on, whose operating system (OS) is set to a different language, navigates to that page with their screen reader turned on
+The screen reader would attempt to read the page in the language that is defined by the lang attribute on the page, but the supporting element information ("button", "link", etc) is read out in the language that is set by the operating system.
+
+To see what happens when this information does not match, we created a new Ember application and created four buttons:
+
+```html
+<button type="button">Click Me</button>
+<button type="button">点击我</button>
+<button type="button">Haz click en mi</button>
+<button type="button">Нажми на меня</button>
+```
+
+No Language Defined:
+
+If no lang attribute is set for the page or the parts, the screen reader defaults to the operating system (OS) language. It reads Spanish in an English accent, and the button element was also still read in English. For the Chinese and Russian letters, it spelled out the letters (i.e., "Cyrillic Letter E")
+
+Language Defined:
+
+We then changed the lang attribute value and listened to these buttons in Chinese(zh), Spanish(es) and Russian(ru). Here's what happened:
+
+- in each case, the announcer's voice changed for the content 
+- since the OS wass set to English, the supporting element information that the assistive tech (AT) announces was in the OS language (English)
```

Comment:
```suggestion
- since the OS was set to English, the supporting element information that the assistive tech (AT) announces was in the OS language (English)
```